### PR TITLE
read gauge from googlesheets rather than excel

### PR DESCRIPTION
**Pull request by zackarno**
just replaced reading in excel file with a function to read all tabs of googlesheet.

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -47,3 +47,20 @@ read_shape_zip <- function(path,layer){
 }
 
 
+read_gauge_googlesheets <- function(url=Sys.getenv("GFH_GAUGE_URL")){
+  sns <- sheet_names(ss = url)
+  sn_filt <- sns[sns!="Sheet1"]
```
Comment (caldwellst) [inline R/utils.R:52]: We only need to read in the gauge sheets I think, so also excluding `return_period` and `metadata`?

**Review by caldwellst — APPROVED** [review]
Generally fine, but confused as this seems to be the same thing you are trying to do with service account as well? Do these branches conflict?

**Comment by zackarno** [comment]
> Generally fine, but confused as this seems to be the same thing you are trying to do with service account as well? Do these branches conflict?

yeah this was a step added before the service account was set up. When we merge the `gsheet-service-acct-auth` branch it just adds a JSON authentication step. It should not conflict

File: R/utils.R (hunk 1 above)
Comment (zackarno) [reply R/utils.R:52]: we should keep metadata as it is the only tab with spatial information which we use. True that loading the `return period` tab is redundant, but I'd like to have one target at least in these early stages that just loads the entire workbook and we an filter from there. Having the`return period` data.frame in that format could even end up saving a wrangling step so I don't think it's to negative to leave in at this stage

**Pull request merged**